modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-test/src/testIntegration/java/com/liferay/headless/commerce/admin/catalog/resource/v1_0/test/ProductConfigurationListAccountGroupResourceTest.java

```diff
@@ -124,12 +107,30 @@ public void testDeleteProductConfigurationListAccountGroup()
 		Assert.assertEquals(0, page.getTotalCount());
 	}
 
-	@Ignore
+	@Override
```

Comment:
question: what does wrong the "base" test that we need to override?

Reply:
this 

	@Test
	public void testGraphQLDeleteProductConfigurationListAccountGroup()
		throws Exception {

		Assert.assertTrue(false);
	}


https://github.com/CrescenzoRegaLF/liferay-portal/blob/bed19a64b70693447813d6742f994c2dacfb3c85/modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-test/src/testIntegration/java/com/liferay/headless/commerce/admin/catalog/resource/v1_0/test/BaseProductConfigurationListAccountGroupResourceTestCase.java#L220